### DISCUSSION TITLE
拉面！

### DISCUSSION THREAD

**Discussion by MQZCZBCK** [share121/inter-knot#1319] [其他]
六分街拉面馆的面太好吃辣！！

**Comment by lllliufeng** [comment]
附议

**Comment by MQZCZBCK** [comment]
豪赤！

**Comment by hpp326** [comment]
![Screenshot_20250122_180233](https://github.com/user-attachments/assets/bf10d9ff-1db0-4dd7-8778-72fa857835de)
确实好吃（之前伦敦绝区零联动面馆里的面）

**Reply by MQZCZBCK** [reply]
哇，看着就好吃<img src="https://img-static.mihoyo.com/communityweb/upload/fa9678e6d88d9f7e526c26d418cd9400.png" alt="该吃饭了" width="75" >

**Comment by sakura558** [comment]
豪赤😋
